easybuild/easyconfigs/g/gmsh/gmsh-4.5.6-intel-2019b-Python-2.7.16.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '4.5.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://gmsh.infio'
```

There is a typo in the URL
```suggestion
homepage = 'https://gmsh.info'
```